README.md

```diff
@@ -169,7 +169,7 @@ The above run-book could be easily automated and integrated into your CI/CD pipe
 
 <a name="lifecycle"></a>
 
-_This section assumes that you have successfully deployed a local environment for testing SNS decentralization and validated your setup by creating an SNS instance for the example dapp (shipped with `sns-testing`)._
+_This section assumes that you have successfully deployed a local environment for testing SNS decentralization and validated your setup by creating an SNS instance for the example dapp (shipped with `sns-testing`). Make sure you ran `./cleanup.sh`_
```

Comment:
If it's always needed (and safe) to run `./cleanup.sh` before proceeding with the instructions in this section, I suggest adding `./cleanup.sh` as the 0-th step, rather than to this comment.

Reply:
I'd drop the paths `candid/assets.did`, `candid/nns-*`, and `candid/sns_*` from `cleanup.sh`. Then it's safe and also needed to run `cleanup.sh` if you want to run SNS lifecycle without redeploying the local replica instance (restarting `dfx start --clean` and rerunning `setup_locally.sh`). Note that we already say to run `cleanup.sh` after you're done testing, but it's not clear that you need to do it for every SNS lifecycle testing.

Reply:
I initally assumed its's a prerequisite due to the wording here

> This section assumes that you have successfully deployed a local environment for testing SNS decentralization and validated your setup by creating an SNS instance for the example dapp (shipped with sns-testing).